apps/api/src/form/form.service.ts

```diff
@@ -24,15 +26,22 @@ export class FormService {
       confirmationId,
     } as Partial<FormEntity>);
 
+    if (!newForm.version) {
+      newForm.version = '1.0.0';
+    }
     const savedForm = await this.formRepository.save(newForm);
-
-    const notifiedForm = await this.sendMail(savedForm);
-
-    return notifiedForm;
+    this.logger.log(`Saved form with id ${savedForm.id}`);
+    if (savedForm.payload?.contactInformation.email) {
```

Comment:
@arranfw @hitankar-freshworks @dbayly-freshworks @vyasworks 
Any improper payload can break API, so this defensive code is introduce .... but no effect if validation enables in system

Reply:
I'd be comfortable with a TODO here, we don't want this staying past go-live